models/issue_tracked_time.go

```diff
@@ -24,7 +25,7 @@ type TrackedTime struct {
 
 // AfterLoad is invoked from XORM after setting the values of all fields of this object.
 func (t *TrackedTime) AfterLoad() {
-	t.Created = time.Unix(t.CreatedUnix, 0).Local()
+	t.Created = time.Unix(t.CreatedUnix, 0).In(setting.UILocation)
```

Comment:
Why have we left the `Created` field in the `TrackedTime` struct?

Reply:
It has one `json:"created"` tag here. I should invistigate where referenced it. Maybe send a PR later.